Blackfynn/blackfynn-python
# Pre-Release 2.4.4

# Description

The purpose of this PR is to fix the Makefile to correctly build the latest documentation folder. 

## Github Issue

Closes #55 


## Type of change



# How Has This Been Tested?

* `cd` into the `docs` folder.
* `make html`
* Confirm that a `latest` folder already exists. If not, copy the most recent semver folder and rename it to `latest`
* `make html-latest`
* Open `docs/_build/html/latest/index.html` file and verify that the latest version found in this file matches the version `blackfynn/__init__.py`

# Checklist:
